src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -519,8 +531,16 @@ public void delete( String name ) {
                 .executeUpdate();
     }
     
-    public Set<Setting> listAll() {
-        return new HashSet<>(em.createNamedQuery("Setting.findAll", Setting.class).getResultList());
+    public Map<String, String> listAll() {
```

Comment:
Maybe listAllSettings?

Reply:
Actually methods in this class do not follow convention of postfixing them with settings so I'm not sure if we should break this convention